docs/chaos-engineering/chaos-faults/aws/ec2-stop-by-id.md

```diff
@@ -152,7 +152,7 @@ spec:
           value: 'instance-1'
         # region for the ec2 instance
         - name: REGION
-          value: 'us-east-1'
+          value: '<region for EC2_INSTANCE_ID>'
```

Comment:
Why did we revert this?

Reply:
I haven't reverted this. It changed after I ran embedmd command - let me update the manifest